Firmware/Marlin_main.cpp

```diff
@@ -631,20 +631,17 @@ void crashdet_detected(uint8_t mask)
 
 void crashdet_recover()
 {
-  if (!print_job_timer.isPaused()) crashdet_restore_print_and_continue();
-  if (lcd_crash_detect_enabled()) tmc2130_sg_stop_on_crash = true;
+	if (!print_job_timer.isPaused()) crashdet_restore_print_and_continue();
```

Comment:
Change the line 620 `uint8_t yesno = lcd_show_yes_no_and_wait(false);` to `uint8_t yesno = lcd_show_yes_no_and_wait(false, LCD_LEFT_BUTTON_CHOICE);`
so RECOVER CRASH default is YES.
I think an accidental knob click should try to recover the crash instead of killing it.